test/runtime/runtime_plugins_test.go

```diff
@@ -0,0 +1,370 @@
+// Copyright 2018 The OPA Authors.  All rights reserved.
+// Use of this source code is governed by an Apache2
+// license that can be found in the LICENSE file.
+
+// +build linux,cgo darwin,cgo
+
+// This exists in a separate testing package for the following reason: https://github.com/golang/go/issues/17928
+
+package runtime
```

Comment:
The top-level test directory has been for integration tests that exercise OPA from the outside. I think it would be better if this test was kept closer to the implementation. E.g., move it to `opa/internal/sharedobjects/test`.